spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -279,3 +281,26 @@ The following figure shows an overview of the Consumer Initiated Slashing operat
   > **Note**: As a consequence of slashing (and potentially jailing) `V`, the Staking module updates accordingly `V`'s voting power. This update MUST be visible in the next VSC provided to the consumer chains.  
 
 For a more detailed description of Consumer Initiated Slashing, take a look at the [technical specification](./technical_specification.md#consumer-initiated-slashing).
+
+### Reward Distribution
+[&uparrow; Back to Outline](#outline)
+
+In the context of single-chain validation, the *Distribution module*, i.e., a module of the ABCI application, handles the distribution of rewards (i.e., block production rewards and transaction fees) to every validator account based on their total voting power; 
+these rewards are then further distributed to the delegators. 
+For an example, take a look at the [Distribution module documentation](https://docs.cosmos.network/v0.44/modules/distribution/) of Cosmos SDK.
+
+At the beginning of every block, the rewards for the previous block are pooled into a distribution module account. 
+The Reward Distribution operation of CCV enables every consumer chain to transfer a fraction of these rewards to the provider chain. 
+The operation consists of two steps that are depicted in the following figure:
+
+![Reward Distribution](./figures/ccv-distribution-overview.png?raw=true)
+
+- At the beginning of every block on the consumer chain, a fraction of the rewards are transferred to an account on the consumer CCV module.
+- At regular intervals (e.g., every `100` blocks), the consumer CCV module sends the accumulated rewards to the distribution module account on the provider chain through an IBC token transfer packet (as defined in [ICS 20](../ics-020-fungible-token-transfer/README.md)). 
+  Note that the IBC transfer packet is sent over a separate unordered channel. 
```

Comment:
This may add some delay / asynchrony compared to the single chain case. For instance, after I undelegate I may still receive fees that I earned before I undelegated. As this is different from the single case system, it may make sense to make this explicit here.